### DISCUSSION TITLE
Old sounds for guns

### DISCUSSION THREAD

**Discussion by ocewot** [danielkrupinski/Osiris#3235] [Ideas]
my suggestion is adding an option to enable the old gun sounds from before the sound updates. i think the old m4a1s sound is way better and sounds amazing same with the USP it would be awesome to have an option to use those again :)

**Comment by aoqia194** [comment]
I agree. This can be done easily. If this gets enough upvotes, I will implement it and then ask Daniel to merge my commit.

**Comment by Nep-Timeline** [comment]
use migi
